test/TimeoutTests/Python/.gitignore

```diff
@@ -0,0 +1,130 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
Should this .gitignore be included at a higher-level, instead of being specific to just this test app?

Reply:
I think it is fine to have it for the test app specifically since the test app is a different type of project (python) than the rest of the code.